chapters/packages.tex

```diff
@@ -290,6 +290,7 @@ \section{External Resources}\label{external-resources}
 
 \begin{nonnormative}
 The use of a trailing segment separator is recommended when the resource is a directory and the file-system path will be prepended to relative file paths within the directory.
+If possible use URIs for specific files or specific sub-directories instead of appending relative paths to a generic URI such as \filename{modelica:/A/Resources/} as the latter creates a dependency on the entire directory.
```

Comment:
Comma for readability:
```suggestion
If possible, use URIs for specific files or specific sub-directories instead of appending relative paths to a generic URI such as \filename{modelica:/A/Resources/} as the latter creates a dependency on the entire directory.
```